adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.m

```diff
@@ -63,22 +106,51 @@ + (GADVersionNumber)adapterVersion {
 
 - (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
                    completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding banner ad.
+    [BUAdSDKManager setCoppa:adConfiguration.childDirectedTreatment.integerValue];
+    _bannerRenderer = [[GADPangleRTBBannerRenderer alloc] init];
+    [_bannerRenderer renderBannerForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
 - (void)loadInterstitialForAdConfiguration:(GADMediationInterstitialAdConfiguration *)adConfiguration
                          completionHandler:(GADMediationInterstitialLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding interstitial ad.
-}
-
-- (void)loadNativeAdForAdConfiguration:(GADMediationNativeAdConfiguration *)adConfiguration
-                     completionHandler:(GADMediationNativeLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding native ad.
+    [BUAdSDKManager setCoppa:adConfiguration.childDirectedTreatment.integerValue];
+    _interstitialRenderer = [[GADPangleRTBInterstitialRenderer alloc] init];
+    [_interstitialRenderer renderInterstitialForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
 - (void)loadRewardedAdForAdConfiguration:(GADMediationRewardedAdConfiguration *)adConfiguration
                        completionHandler:(GADMediationRewardedLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding rewarded ad.
+    [BUAdSDKManager setCoppa:adConfiguration.childDirectedTreatment.integerValue];
+    _rewardedlRenderer = [[GADPangleRTBRewardedRenderer alloc] init];
+    [_rewardedlRenderer renderRewardedAdForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
++ (void)setCoppa:(NSInteger)coppa {
+    if (coppa == 0 || coppa == 1) {
+        _coppa = coppa;
+        if (BUAdSDKManager.initializationState == BUAdSDKInitializationStateReady) {
+            [BUAdSDKManager setCoppa:_coppa];
+        }
+    }
+}
+
++ (void)setGDPR:(NSInteger)GDPR {
+    if (GDPR == 0 || GDPR == 1) {
```

Comment:
In iOS, does Pangle SDK take `-1` as the GDPR value too?

That is the case in Android: https://github.com/googleads/googleads-mobile-android-mediation/pull/393/files#r820421421.

Reply:
yes,-1 is default value